README.md

```diff
@@ -760,6 +761,29 @@ Apache Cordova is a popular mobile application development framework using CSS3,
 
 ---
 
+#### permission
+
+> Backward compatibility of [`angular-permission`](https://github.com/Narzerus/angular-permission). Fully featured permission based access control for your angular 2.0 applications.
+
+##### Managing permissions
```

Comment:
There's no reason to add each doc url here. Simply add [ng2-permission](https://github.com/JavadRasouli/ng2-permission) under [Components](https://github.com/AngularClass/awesome-angular#components-1), that should suffice. People can navigate the docs from the component repo.